server/chat-plugins/wifi.ts

```diff
@@ -150,15 +150,15 @@ class Giveaway {
 			for (const [key, value] of mons) {
 				let spriteid = value.spriteid;
 				if (value.cosmeticFormes) {
-					for (const form of value.cosmeticFormes) {
-						if (text.includes(form)) {
-							spriteid += '-' + form.substr(key.length);
+					for (const forme of value.cosmeticFormes.map(toID)) {
+						if (text.includes(forme)) {
+							spriteid += '-' + forme.substr(key.length);
```

Comment:
```suggestion
							spriteid += '-' + forme.slice(key.length);
```

(We're deprecating `substr`)